src/couch_views/src/couch_views_active_tasks.erl

```diff
@@ -0,0 +1,87 @@
+% Licensed under the Apache License, Version 2.0 (the "License"); you may not
+% use this file except in compliance with the License. You may obtain a copy of
+% the License at
+%
+%   http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+% WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+% License for the specific language governing permissions and limitations under
+% the License.
+
+
+-module(couch_views_active_tasks).
+
+
+-export([
+    populate_active_tasks/2
+]).
+
+
+-include("couch_views.hrl").
+-include_lib("couch/include/couch_db.hrl").
+-include_lib("couch_mrview/include/couch_mrview.hrl").
+-include_lib("fabric/include/fabric2.hrl").
+
+
+populate_active_tasks(<<"views">>, Data)  ->
+    #{
+        <<"db_name">> := DbName,
+        <<"ddoc_id">> := DDocId,
+        <<"changes_done">> := ChangesDone,
+        <<"db_uuid">> := DbUUID
+    } = Data,
+    
+    {ok, Db} = try
+        fabric2_db:open(DbName, [?ADMIN_CTX, {uuid, DbUUID}])
+    catch error:database_does_not_exist ->
+        {ok, not_found}
+    end,
+
+    {ok, DDoc} = case fabric2_db:open_doc(Db, DDocId) of
+        {ok, DDoc0} ->
+            {ok, DDoc0};
+        {not_found, _} ->
+            {ok, not_found}
+    end,
```

Comment:
A bit worried about this. We could be starting two transactions here for each task item. One for opening the db and other for opening the doc. With a few hundred active tasks and someone say polling `_active_tasks` for monitoring that could add a lot of load to the system just to add `VStamp` and `DbStamp` to the data.

Reply:
Seems we have two issues
1) exceeding the 100 kb limit with JobData
2) too many read requests when active_task processes exist.

The only 3 things we need are really `changes_done`, `view_seq` and `db_seq`. Wonder if adding just those JobData would an okay compromise?

Reply:
oh, `view_seq` already exists so only really just need to add `db_seq`